### PR TITLE
Various documentation fixes before 1.5.0 RC2

### PR DESCRIPTION
**Pull request by philippjfr**

### CODE REVIEW

**Comment by hoxbro** [comment]
https://holoviz-dev.github.io/panel/explanation/comms.html

File: doc/conf.py [hunk 1]
```diff
@@ -82,15 +84,27 @@
     "announcement": announcement_text,
 }
 
-extensions += [
-    'sphinx.ext.napoleon',
-    'nbsite.gallery',
+
+extensions = [
```
Comment (hoxbro) [inline doc/conf.py:88]: Wouldn't it be easy to remove the one you don't like?

File: pixi.toml [hunk 2]
```diff
@@ -177,12 +177,15 @@ test-type = 'mypy panel'
 [feature.doc.activation.env]
 MOZ_HEADLESS = "1"
 PANEL_IPYWIDGET = "1"
+OBJC_DISABLE_INITIALIZE_FORK_SAFETY = "YES"
```
Comment (hoxbro) [inline pixi.toml:180]: Still needed?

Comment (philippjfr) [reply pixi.toml:180]: Yes

File: doc/conf.py (hunk 1 above)
Comment (philippjfr) [reply doc/conf.py:88]: If I was just removing one of them yes, but with two I find this nicer/more explicit.

**Pull request merged**